doc/_static/extract_api_implementation.py

```diff
@@ -0,0 +1,96 @@
+
+if response['result']['status_code'] == SUCCESS:
+
+    # You received an id that you can use to poll the server to get the result of the ocr when it will be ready
+    document_id = response['result']['document_id']
+    params = {
+        'version': version,
+        'document_ids': [document_id],  # you can request the results of multiple documents at once if wanted
+    }
+    document_id = str(document_id)
+    response = jsonrpc("/iap/invoice_extract/get_results", params)
+    i = 0
+    with open('out_' + str(i) + '.txt', 'w') as outfile:
+        json.dump(response, outfile, indent=2)
+        i += 1
+    while response['result'][document_id]['status_code'] == NOT_READY:  # 1 is the status code indicating that the server is still processing the document
+        print("Still processing... Retrying in 5 seconds")
+        time.sleep(5)
+        response = jsonrpc("/iap/invoice_extract/get_results", params)
+        with open('out_' + str(i) + '.txt', 'w') as outfile:
+            json.dump(response, outfile, indent=2)
+            i += 1
+    print(response)
+    if response['result'][document_id]['status_code'] != SUCCESS:
+        print(response['result'][document_id]['status_msg'])  # if it isn't a success, print the error message
+    else:
+        document_results = response['result'][document_id]['results'][0]
+        print("Total:", document_results['total']['selected_value']['content'])
+        print("Subtotal:", document_results['subtotal']['selected_value']['content'])
+        print("Invoice id:", document_results['invoice_id']['selected_value']['content'])
+        print("Date:", document_results['date']['selected_value']['content'])
+        print("...")
+
+        params = {
+            'values': {
+                'total': {'content': 100.0},
+                'subtotal': {'content': 100.0},
+                'global_taxes': {'content': []},
+                'global_taxes_amount': {'content': 0.0},
+                'date': {'content': '2020-09-25'},
+                'due_date': {'content': '2020-09-25'},
+                'invoice_id': {'content': '79'},
+                'partner': {'content': 'twinnta'},
+                'VAT_Number': {'content': 'BE23252248420'},
+                'currency': {'content': 'USD'},
+                'merged_lines': False,
+                'invoice_lines': {'lines': [{'description': 'Total TVA ',
+                                             'quantity': 1.0,
+                                             'unit_price': 100.0,
+                                             'product': False,
+                                             'taxes_amount': 0.0,
+                                             'taxes': [],
+                                             'subtotal': 100.0,
+                                             'total': 100.0}]
+                                  }
+            },
+            'document_id': document_id
+        }
```

Comment:
The purpose of this script is to show how the API can be used. The script should not be used as it is and should be adapted for each specific case. Since we are in integration mode in this script (because of the token used), the result obtained with get_result is also hardcoded and is not specific to the document sent in the /parse call. The validation content could be a mixed of hardcoded value (as now) and values from the result of the /get_result call to show the validation values don't have to come especially from the get_value result.